activerecord/CHANGELOG.md

```diff
@@ -24,6 +24,11 @@
 
 ## Rails 5.0.0 (June 30, 2016) ##
 
+*   AM::Dirty's changed? methods now return an actual boolean, never `nil`, as in 4.2.
```

Comment:
Thanks for the feedback Kasper. I can make all of those changes, except "move this to the correct changelog" which I don't understand. See comments above about branches.